doc/classes/@GlobalScope.xml

```diff
@@ -1092,7 +1092,7 @@
 			<param index="3" name="ostart" type="float" />
 			<param index="4" name="ostop" type="float" />
 			<description>
-				Maps a [param value] from range [code][istart, istop][/code] to [code][ostart, ostop][/code]. See also [method lerp] and [method inverse_lerp]. If [param value] is outside [code][istart, istop][/code], then the resulting value will also be outside [code][ostart, ostop][/code]. If this is not desired, use [method clamp] on the result of this function.
+				Maps a [param value] from range [code][istart, istop][/code] to [code][ostart, ostop][/code]. See also [method lerp] and [method inverse_lerp]. If [param value] is outside [code][istart, istop][/code], then the resulting value will also be outside [code][ostart, ostop][/code]. If this is not desired, use [method clamp] on the result of this function. If [param istart] and [param istop] are the same, the function will return NaN, use [method is_nan] to guard against this.
```

Comment:
Actually for `istart == istop` case it can also return `INF` or `-INF` (these and `NaN` can be checked against with `is_finite` method).

Basically:
- `value < istart == istop` then result is `lerp(ostart, ostop, -INF)`,
- `value == istart == istop` then result is `lerp(ostart, ostop, NaN)`,
- `value > istart == istop` then result is `lerp(ostart, ostop, INF)`.